src/home_robot_hw/home_robot_hw/ros/msg_numpy.py

```diff
@@ -101,55 +102,13 @@ def image_to_numpy(msg):
 
 # @converts_from_numpy(Image)
 def numpy_to_image(arr, encoding):
-    if not encoding in name_to_dtypes:
```

Comment:
again this is a weird amount of code getting deleted in a refactor - what's happening here?

Reply:
The exact same `numpy_to_image` function was declared two times -- probably a mistake during copy-paste.